_posts/2020-06-30-june-2020-roundup.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "What we've been reading in June"
+author: tyler
+tags: [roundup]
+---
+
+<!-- excerpt start -->
+
+Here are the articles, videos, and tools that we've been excited about this
+June.
+
+<!-- excerpt end -->
+
+We hope you enjoy these links, and we look forward to hearing what you've been
+reading in the comments.
+
+## Articles & Learning
+
+- [Embedded Systems Security and TrustZone](https://embeddedsecurity.io/) by
+  Dimitrios Slamaris<br>Incredible resource published today on security best
+  practices on an ARM Cortex-M, but starts from the basics with an intro into C
+  programming, GNU tools, Cortex-M architecture, and bare-metal firmware
+  tutorials. Really, really good. Bookmark this.
+- [Fast Execute-Only Memory for Embedded Systems](https://arxiv.org/abs/2006.00076)
+  by Zhuojia Shen and John Criswell of Cornell University<br>A paper about
+  preventing remote code disclosure attacks with a clever use of the ARM
+  Cortex-M MPU and DWT to implement Execute-Only-Memory (XOM).
+- nRF52 Debug Resurrection (APPROTECT Bypass) -
```

Comment:
"A complete compromise of the flash readout protection on nRF52 series microcontrollers using simple voltage glitching techniques"